README.md

```diff
@@ -22,6 +22,14 @@ examples.
 For more details on how to setup and use the group messaging example please see
 the [README](app/modules/group-messaging/README.md) in that module.
 
+### Screen Sharing
+
+Initial screen sharing support included on Chrome support. Support for other browsers will be added
```

Comment:
I bet it's supposed to read "includes only Chrome support." That's still awkward, though. I'll reword.